OSVectorTileAPI/OpenLayers/map.js

```diff
@@ -27,7 +27,9 @@ function setupLayer() {
     message.classList.remove("warning");
     message.textContent = 'To view the map, please enter a valid API key.';
     instructions.classList.add("hidden");
-
+    
+    // Our Vector Tile Service requires the main Capabilities url, the style (our default one in this case) and due to the style elements used the sprite file
```

Comment:
Our application requires...

it's not the service that needs this stuff, it's the app we are building